cmd/minishift/cmd/root.go

```diff
@@ -285,10 +288,19 @@ func initConfig() {
 		constants.ProfileName = profile
 		constants.ConfigFile = constants.MakeMiniPath("profiles", profile, "config", "config.json")
 	}
+
+	// Here we first check if anything in the global config fi
```

Comment:
This comment does look correct here. Suggestion `Initializing the global config file with Viper`